.github/workflows/rust-ci.yaml

```
@@ -128,3 +128,55 @@ jobs:
           YARA_INCLUDE_DIR: ${{ github.workspace }}/.yara/${{ needs.yara-test.outputs.include-dir }}
           YARA_LIBRARY_PATH: ${{ github.workspace }}/.yara/${{ needs.yara-test.outputs.library-path }}
         run: cargo test --no-fail-fast
+
+  docs-build:
+    needs: [yara]
+    runs-on: ubuntu-latest
```

As with the other jobs,
```suggestion
    runs-on: ubuntu-22.04
```

---

99e2331ab6dce0b6484eaaba36508d91aa9743b0